src/main/java/gov/nih/nci/evs/api/service/MetadataServiceImpl.java

```diff
@@ -53,17 +52,17 @@ public class MetadataServiceImpl implements MetadataService {
    * @throws Exception the exception
    */
   @Override
```

Comment:
Yeah, we just don't need caching here there's one place we need it for performance while doing RDF load, but otherwise - I think these were being cached for reasons that no longer apply (not performance issues anymore and we just never removed them) .  Good finds.